src/pages/workflows/WorkflowList.test.tsx

```diff
@@ -0,0 +1,560 @@
+import { delay } from '@terra-ui-packages/core-utils';
```

Comment:
These Typescript tests continue to feel excessive to me, and tightly coupled to the specific implementation. Is there a way we can cover the 80% most valuable functionality with 20% of the code? IMO we should not be testing every last little behavior.